versioned_docs/version-v3.0.0/node/node-operations.md

```
@@ -0,0 +1,220 @@
+</TabItem>
+<TabItem value="docker_compose" label="Docker Compose">
+
+### Start the HOPRd node
+
+1. Go to your **compose** folder.
+
+2. Use the profiles feature to start only the **hoprd** profile associated with the hopr node. 
+
+    Run the following command: 
+    
+    ```
```

markdownlint-cli2 (0.17.2)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)